adcc/backends/test_psi4.py

```diff
@@ -111,9 +111,18 @@ def template_rhf_h2o(self, basis):
 
         # Test dipole
         mints = psi4.core.MintsHelper(wfn.basisset())
-        ao_dip = [np.array(comp) for comp in mints.ao_dipole()]
+        ao_dip = [-1.0 * np.array(comp) for comp in mints.ao_dipole()]
```

Comment:
Really? How did we miss that before?

Reply:
Because apparently the `assert_allclose_signfix` was used... 😦 Now it's correct though...